src/toil_scripts/adam_pipeline/adam_preprocessing.py

```diff
@@ -36,11 +36,12 @@
 
 import yaml
 from toil.job import Job
```

Comment:
My comment still stands. PEP8 wants a blank line between the standard library imports and third-party imports. I agree with you though that it looks nicer your way.